README.md

```diff
@@ -387,6 +392,30 @@ This will record each token issued in your database, confirm it is still valid o
 
 For more in-depth documentation please see the [GuardianDb README](https://github.com/ueberauth/guardian_db/blob/master/README.md)
 
+## Best testing practices
+
+### How to add the token to a request (the Phoenix way)
+
+```elixir
+defmodule HelloWeb.AuthControllerTest do
+  use HelloWeb.ConnCase
+  import HelloWeb.Guardian
+
+ test "GET /auth/me", %{conn: conn} do
+    user = insert(:user) // See https://github.com/thoughtbot/ex_machina
```

Comment:
This needs to be `# See https://github.com/thoughtbot/ex_machina`

Reply:
Oh I will correct that :)